lib/connection.ts

```diff
@@ -376,6 +376,13 @@ export class Connection extends Entity {
     return result;
   }
 
+  /**
+   * Remove all the sessions from the internal map.
+   */
+  removeAllSessions(): void {
+    return this._connection.remove_all_sessions();
```

Comment:
Lets check for `this._connection` before calling `remove_all_sessions` on it.

Since `remove_all_sessions` doesn't return anything, we can skip the "return" keyword